app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_file = command.split()
+    if list_file == 3 and list_file[1] != list_file[2]:
```

Comment:
Bug: The condition `list_file == 3` is incorrect. It seems you want to check if the length of the list is 3. You should use `len(list_file) == 3` instead.